templates/index.html

```diff
@@ -15,13 +15,13 @@ <h1 class="u-off-screen">Snapcraft</h1>
     {% endif %}
     <div class="col-12">
       <h2>A Universal App Store for Linux</h2>
-      <p>Deliver and update your app on any Linux distribution - for desktop, cloud,  and Internet of Things.</p>
+      <p>Deliver and update your app on any Linux distribution - <br class="u-hide--small" />for desktop, cloud,  and Internet of Things.</p>
```

Comment:
This hanging dash looks a bit weird IMO, but maybe someone better at typography can judge it.